src/main/webapp/WEB-INF/views/admin/project/src-js.jsp

```diff
@@ -837,9 +837,11 @@ var src_grid = {
 										
 										srcList.jqGrid("setCell", rowid, "licenseName", licenseName);
 										fn_grid_com.saveCellData(srcList.attr("id"), rowid, "licenseName", licenseName, null, null);
-										srcList.jqGrid('saveRow',rowid);
-										
-										fn_grid_com.mvOssPage(srcList, rowid);
+                                        currentOssName = e.value;
```

Comment:
Good! three commits are combined into one commit and signed-off.
But it is recommened to fix indentation 840~844 lines.

Reply:
Thank you for your good point.